app/code/core/Mage/Adminhtml/Block/Sales/Items/Abstract.php

```diff
@@ -580,17 +577,11 @@ public function canShipPartiallyItem($order = null)
             $order = Mage::registry('current_shipment')->getOrder();
         }
         $value = $order->getCanShipPartiallyItem();
-        if (!is_null($value) && !$value) {
-            return false;
-        }
-        return true;
+        return !(!is_null($value) && !$value);
     }
 
     public function isShipmentRegular()
     {
-        if (!$this->canShipPartiallyItem() || !$this->canShipPartially()) {
-            return false;
-        }
-        return true;
+        return !(!$this->canShipPartiallyItem() || !$this->canShipPartially());
```

Comment:
Apply DeMorgan's Law: `!(A || B) == (!A && !B)`
```suggestion
        return $this->canShipPartiallyItem()  && $this->canShipPartially();
```